### ISSUE TITLE
computeStateDisplay does not utilize translation_key

### ISSUE THREAD

**Issue by mill1000** [benct/lovelace-multiple-entity-row#290]
The current implementation of `computeStateDisplay` does not utilize an entity's translation_key which can result in unlocalized states.
e.g.
As of Core 2023.4.0, The transmission status sensor is capitalized when used in Entities, but not in multiple-entity-row
![image](https://user-images.githubusercontent.com/6924622/231301617-ad17b9cb-4e3e-452c-8160-0176715d53d7.png)

The current implementation:
https://github.com/benct/lovelace-multiple-entity-row/blob/e2d250821306616291f8ea6e695d79a9cb1670bd/src/lib/compute_state_display.js#L106-L114

Possible correct implementation from the frontend.
```ts
return (
    (entity?.translation_key &&
      localize(
        `component.${entity.platform}.entity.${domain}.${entity.translation_key}.state.${state}`
      )) ||
    // Return device class translation
    (attributes.device_class &&
      localize(
        `component.${domain}.entity_component.${attributes.device_class}.state.${state}`
      )) ||
    // Return default translation
    localize(`component.${domain}.entity_component._.state.${state}`) ||
    // We don't know! Return the raw state.
    state
  );
````

**Comment by rtasarz** [comment]
After some tweaking replacement code indeed works. I have committed changes to my fork, but I need to find some time to check if adapted `entity?.translation_key` condition is changed correctly before preparing pull request.

**Comment by mill1000** [comment]
I made a PR as well. #293

**Comment by mill1000** [comment]
Closed via #293
